.github/workflows/run_hac_test.yml

```diff
@@ -0,0 +1,43 @@
+name: HAC on KIND cluster Test
+
+# Run on each new PR and each new push to existing PR
+on: [push, pull_request]
+
+env:
+  ACTIONS_ALLOW_UNSECURE_COMMANDS: "true"
+  GO111MODULE: "on"
+  GOPROXY: "https://proxy.golang.org"
+  KUBERNETES_VERSION: "v1.17.3"
```

Comment:
Why Do we need this Env variable?

Reply:
Probably not, removed. 10x, @liranmauda !